### PR TITLE
en: update doc for gke deployment

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Update doc for GKE deployment.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/42
- Other reference link(s):

### CODE REVIEW

File: en/deploy-on-gcp-gke.md [hunk 1]
```diff
@@ -154,29 +155,76 @@ This section describes how to deploy a TiDB cluster.
...
+    ```
+    sed 's/CLUSTER_NAME/<tidb-cluster-name>/g' db.yaml.example > db.yaml
+    sed 's/CLUSTER_NAME/<tidb-cluster-name>/g' db-monitor.yaml.example > db-monitor.yaml
+    ```
+
+    To complete the CR file configuration, refer to [TiDB Operator API documentation](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.html) and [Configure Cluster using TidbCluster](configure-cluster-using-tidbcluster.md).
+
+    > **Note:**
+    >
+    > * Make sure the numbers of nodes in PD, TiKV, and TiDB are the same as the `replicas` field in `db.yaml`. Note that in the Regional cluster, the number of nodes actully created is `pd_count/tikv_count/tidb_count` * 3.
```
Comment (TomShawn) [inline en/deploy-on-gcp-gke.md]:
```suggestion
    > * Make sure the number of PD nodes, TiKV nodes, or TiDB nodes is the same as the value of the `replicas` field in `db.yaml`. Note that in the Regional cluster, the number of nodes to actually create is `pd_count` * `3`, `tikv_count` * `3`, or `tidb_count` * `3`.
```

File: en/deploy-on-gcp-gke.md [hunk 2]
```diff
@@ -154,29 +155,76 @@ This section describes how to deploy a TiDB cluster.
...
+    kubectl --kubeconfig credentials/kubeconfig_<gke_name> create -f db-monitor.yaml -n <namespace>
     ```
 
 ## Access the TiDB database
 
 After `terraform apply` is successful executed, perform the following steps to access the TiDB cluster. Replace the `<>` section with the output of running `terraform apply` above.
 
-1. Connect to the bastion machine by using `ssh`.
+1. Get the IP address of the TiDB Internal LoadBalancer:
+
```
Comment (TomShawn) [inline en/deploy-on-gcp-gke.md]:
```suggestion

{{< copyable "shell-regular" >}}

```

File: en/deploy-on-gcp-gke.md [hunk 3]
```diff
@@ -186,7 +234,8 @@ After `terraform apply` is successful executed, perform the following steps to a
 
     > **Note:**
     >
-    > You need to install the MySQL client before you connect to TiDB via MySQL.
+    > You need to install the MySQL client before you connect to TiDB via MySQL. If you use CentOS, install by `sudo yum install -y mysql`.
```
Comment (TomShawn) [inline en/deploy-on-gcp-gke.md]:
```suggestion
    > You need to install the MySQL client before you connect to TiDB via MySQL. If you use CentOS, install the client by executing `sudo yum install -y mysql`.
```

File: en/deploy-on-gcp-gke.md [hunk 4]
```diff
@@ -186,7 +234,8 @@ After `terraform apply` is successful executed, perform the following steps to a
...
+    > `<tidb_ilb_ip> is the IP address of the Internal LoadBalancer acquired before.
```
Comment (TomShawn) [inline en/deploy-on-gcp-gke.md]:
```suggestion
    > `<tidb_ilb_ip> is the IP address of the Internal LoadBalancer acquired in step 1.
```

File: en/deploy-on-gcp-gke.md [hunk 5]
```diff
@@ -283,54 +295,42 @@ An instance of a `tidb-cluster` module corresponds to a TiDB cluster in the GKE
...
-    {{< copyable "" >}}
-
-    ```hcl
-    output "how_to_connect_to_example_tidb_cluster_from_bastion" {
-    value = module.example-tidb-cluster.how_to_connect_to_tidb_from_bastion
-    }
-    ```
-
-    This above configuration enables this script to print out the exact command used to connect to the TiDB cluster.
+    > - The total number of nodes actually created for each component = the number of nodes in the configuration file * the number of Availability Zones in the Region. The number of Regional clusters is 3 by default.
```
Comment (TomShawn) [inline en/deploy-on-gcp-gke.md]:
```suggestion
    > - The total number of nodes actually to create for each component = the number of nodes in the configuration file * the number of Availability Zones in the Region. The number of Regional clusters is `3` by default.
```

File: en/deploy-on-gcp-gke.md [hunk 6]
```diff
@@ -133,7 +133,8 @@ This section describes how to deploy a TiDB cluster.
 
     > **Note:**
     >
-    > The number of worker nodes created depends on the number of Availability Zones in the specified Region. Most Regions have 3 zones, but `us-central1` has 4 zones. See [Regions and Zones](https://cloud.google.com/compute/docs/regions-zones/) for more information and see the [Customize](#customize) section on how to customize node pools in a regional cluster.
+    > * The Regional cluster is created by default, which creates nodes in the 3 Availability Zones. For example, if you configure `pd_count = 1`, 3 nodes are actually created for PD. You can specify the Availability Zones by configuring `node_locations`, or create the Zonal cluster by configuring `location`. See the example in `examples/` for details.
```
[the remark this answers is not in the document]
Comment (cofyc) [reply en/deploy-on-gcp-gke.md]: yes, it's correct.

**Comment by ran-huang** [comment]
@TomShawn PTAL

**Pull request merged**